src/hooks/useFilterPossiblyMaliciousPositions.ts

```diff
@@ -0,0 +1,58 @@
+import { Interface } from '@ethersproject/abi'
+import { Token } from '@uniswap/sdk-core'
+import ERC20ABI from 'abis/erc20.json'
+import { Erc20Interface } from 'abis/types/Erc20'
+import { useMultipleContractSingleData } from 'lib/hooks/multicall'
+import { useMemo } from 'react'
+import { PositionDetails } from 'types/position'
+import { hasURL } from 'utils/urlChecks'
+
+import { useDefaultActiveTokens } from './Tokens'
```

Comment:
Is this a typo or expected that a file in `hooks` folder (not a component?) is uppercase?

Reply:
I agree this is weird and should be changed